src/unlink.js

```diff
@@ -89,10 +89,14 @@ module.exports = function unlink(config, args) {
   unlinkDependencyIOS(project.ios, dependency, packageName);
 
   const allDependencies = getDependencyConfig(config, getProjectDependencies());
+  const otherDependencies = filter(allDependencies, ({name}) => name !== packageName);
```

Comment:
Ok makes sense. I will fix that. The `eslintrc` allows it, so should I remove it from there as well so nobody else gets caught by this?